AzureMonitorAgent/ama_tst/modules/install/check_os.py

```diff
@@ -0,0 +1,107 @@
+import platform
+
+from error_codes import *
+from errors      import error_info
+from helpers     import find_vm_bits, find_vm_distro
+
+supported_dists_x86_64 = {'redhat' : ['7', '8'], # Rhel
```

Comment:
i assume here we will keep track of the latest supported distro list. however customers may run the TST for older versions of AMA. is there a way to warn the customer if they have installed an old AMA version on a new distro that wasn't supported at the time?

Reply:
same for old versions that didn't support aarch64

Reply:
TST is bundled with AMA at this time, so I assume TST will always check os that is supported in this AMA version. Under which condition customers may run the latest TST for older versions of AMA?

Reply:
I don't think our documentation or somewhere specifies the range of distros that one AMA version supports. I think a good way is to add a warning message that older AMA versions may not support but I can't tell if it actually supports or not in TST. Is simply adding a warning message an acceptable solution?

Reply:
Since this is an unlikely scenario, can we keep this check as it is now? In the next part we check AMA version and suggest updating to the latest version to avoid any problems. I'm afraid we'll confuse users saying that current TST cannot check older AMA supported distros

Reply:
yep agreed that it is pretty unlikely